scoper.inc.php

```diff
@@ -117,6 +117,8 @@ function( $file_path, $prefix, $contents ) {
 				$contents = str_replace( '"Google_', '"' . $prefix . '\Google_', $contents );
 			}
 			if ( false !== strpos( $file_path, 'apiclient-services-subscribewithgoogle' ) ) {
+				// Rewrite 'Class_Name' to Class_Name::class to inherit namespace.
+				$contents = preg_replace( '/\'(Google_[^\']+)\'/', '\\1::class', $contents );
```

Comment:
This looks unrelated but was actually causing the publications within `Google_Service_SubscribewithGoogle_ListPublicationsResponse` to not be parsed into the respective class instances, but was left as a raw array instead.